sources/engine/Stride.Physics/Engine/PhysicsComponent.cs

```diff
@@ -645,6 +645,12 @@ internal void Attach(PhysicsProcessor.AssociatedData data)
             //this is mostly required for the game studio gizmos
             if (Simulation.DisableSimulation)
             {
+                //Give warning if CharacterComponent doesnt have collider shapes
+                if(this is CharacterComponent && ColliderShapes.Count == 0 && ColliderShape == null)
+                {
+                    logger.Warning($"Entity {this} has no physical shapes attatched to CharacterComponent.");
+                }
+
```

Comment:
This makes sense but it is kind of a duplicate of the check further below, might as well just keep one of them and have it above the `if (Simulation.DisableSimulation)` test